### PR TITLE
Plane features + testing

### PR DESCRIPTION
**Pull request by LeonteDenismsgDev**

### CODE REVIEW

File: src/main/java/msg/flight/manager/persistence/repositories/PlaneRepository.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Setter
+@Repository
+public class PlaneRepository{
+    @Autowired
+    MongoTemplate template;
```
Comment (msgIasmina) [inline src/main/java/msg/flight/manager/persistence/repositories/PlaneRepository.java]: private

File: src/main/java/msg/flight/manager/services/planes/PlaneService.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class PlaneService {
+
+    @Autowired
+    PlaneRepository repository;
```
Comment (msgIasmina) [inline src/main/java/msg/flight/manager/services/planes/PlaneService.java]: private

**Review by msgIasmina — CHANGES_REQUESTED** [review]
just privacy setters for some class attributes , nu am pus la tiate clasele am pus doar  la primele  doua  care le-am vazut

File: src/main/java/msg/flight/manager/persistence/repositories/PlaneRepository.java (hunk 1 above)
Comment (LeonteDenismsgDev) [reply src/main/java/msg/flight/manager/persistence/repositories/PlaneRepository.java]: fixed

File: src/main/java/msg/flight/manager/services/planes/PlaneService.java (hunk 2 above)
Comment (LeonteDenismsgDev) [reply src/main/java/msg/flight/manager/services/planes/PlaneService.java]: fixed

**Review by msgIasmina — APPROVED** [review]

**Pull request merged**